File: benchmark/fib/fib.lua
function fib(n)
	if n < 2 then
		return n
	end

	return fib(n - 2) + fib(n - 1)
end

local start = os.clock()

io.write(fib(35).."\n")

io.write(string.format("elapsed: %g\n", os.clock() - start))

File: test/benchmark/fib/fib.py
from __future__ import print_function
import time

def fib(n):
	if n < 2: return n
	return fib(n - 1) + fib(n - 2)

start = time.time()

print(fib(35))

print("elapsed: " + str(time.time() - start))